fix: minor information to docs (#290)
aerogear/offix: docs/ref-advanced.md

---
id: cookbooks
title: Cookbooks
sidebar_label: Offline Cookbook
---

# Offix Cookbook

> This section contains multiple tips and trics for working with Offix.

Reviewer: ```suggestion
This section contains multiple tips and tricks for working with Offix.
```

  isOffline(): boolean;
```

This interface can be used to redefine what being offline means. 

> For example for some use cases developers might decide to use scheduler only when Wifi connection is available.

Reviewer: ```suggestion
For example, for some use cases developers might decide to use scheduler only when Wifi connection is available.
```

> ## Querying data when offiline

Reviewer: ```suggestion
## Querying Data when Offline
```

> To make sure that data will be available when device will be going online we need to query it with the proper 

Reviewer: ```suggestion
To make sure that data will be available when the devices comes back online, we need to query it with the proper 
```

`fetchPolicy`

> We recomend to always use default fetch policy and let offix to control the folow, however for advanced use cases

Reviewer: ```suggestion
We recommend to always use the default fetch policy and to let Offix to control the flow. However, for advanced use cases
```
